tools/notifyServicelogEvent.c

```diff
@@ -32,19 +32,20 @@
 #include <crm/common/attrd_internal.h>
 #include <crm/common/ipc_attrd_internal.h>
 
-typedef enum { STATUS_GREEN = 1, STATUS_YELLOW, STATUS_RED } STATUS;
+typedef enum { STATUS_GREEN, STATUS_YELLOW, STATUS_RED } STATUS;
```

Comment:
was supposed to be mentioned in the commit message but would prefer to leave it alone unless I update for another reason